packages/firebase_vertexai/firebase_vertexai/example/lib/main.dart

```diff
@@ -526,7 +526,7 @@ class MessageWidget extends StatelessWidget {
             decoration: BoxDecoration(
               color: isFromUser
                   ? Theme.of(context).colorScheme.primaryContainer
-                  : Theme.of(context).colorScheme.surfaceContainerHighest,
+                  : Theme.of(context).colorScheme.surfaceVariant,
```

Comment:
If you revert this, the analyse CI should successfully run again

Reply:
done